tests/common/test_models_builder.py

```diff
@@ -1,10 +1,64 @@
 import numpy as np
 import pytest
 
+from doctr.file_utils import CLASS_NAME
 from doctr.io import Document
 from doctr.models import builder
 
 
+def test_documentbuilder_dict_blocks():
+
+    words_per_page = 10
+    num_pages = 2
+
+    # Don't resolve lines
+    doc_builder = builder.DocumentBuilder(resolve_lines=False, resolve_blocks=False)
+    boxes = {CLASS_NAME: np.random.rand(words_per_page, 6)}
+    boxes[CLASS_NAME][:2] *= boxes[CLASS_NAME][2:4]
```

Comment:
After some research, I think you are looking for parametrized tests (`dict` as input and `list` as input). See https://docs.pytest.org/en/6.2.x/parametrize.html

Something like this:
```python
@pytest.mark.parametrize("doc_builder,boxes", [(doc_builder_1, boxes_1),(doc_builder_2, boxes_2)])
def test_documentbuilder_dict_or_list_blocks(doc_builder, boxes):
    # the rest
```
it would be enough for this test

Reply:
Nice ! Done and pushed.